BibBuddy/app/src/main/java/de/bibbuddy/BookBarcodeScannerFragment.java

```diff
@@ -187,6 +194,17 @@ public void closeFragment() {
     fragment.setArguments(createBookBundle());
   }
 
+  private void closeFragmentAfterAdded() {
+    FragmentManager fragmentManager = getParentFragmentManager();
+    if (fragmentManager.getBackStackEntryCount() > 1) {
+      fragmentManager.popBackStack();
+      fragmentManager.popBackStack();
+    } else {
+      requireActivity().onBackPressed();
+      requireActivity().onBackPressed();
```

Comment:
```suggestion
```
Remove double code.

Reply:
This code is not double but unfortunately need to proper function - this will be adressed by the later PR cancelling the manual overwriting of the back button in the whole app.